tap-snapshots/test-02-snapshot-bin-east-commands-create-withUnsupportedFileExtension.js-TAP.test.js

```diff
@@ -0,0 +1,11 @@
+/* IMPORTANT
+ * This snapshot file is auto-generated, but designed for humans.
+ * It should be checked into source control and tracked carefully.
+ * Re-generate by setting TAP_SNAPSHOT=1 and running tests.
+ * Make sure to inspect the output below.  Do not ignore changes!
+ */
+'use strict'
+exports[`test/02-snapshot/bin/east/commands/create/withUnsupportedFileExtension.js TAP > output 1`] = `
+Adapter doesn't provide template '.bruh' source files
```

Comment:
Don't know whether this is good to make this a snapshot, because different adapters may produce different error messages and it is advertised that people writing custom adapters can test them with the tests from this repo...

Reply:
Currently only test/01-migrator tests intend to use with custom adapters so It's good to have this test.